src/main/java/io/github/moulberry/notenoughupdates/util/DiscordMarkdownBuilder.java

```diff
@@ -33,7 +33,19 @@ public DiscordMarkdownBuilder category(String name) {
 	}
 
 	public DiscordMarkdownBuilder append(String key, Object value) {
-		builder.append("[").append(key).append("]").append("[").append(value).append("]").append("\n");
+		if (!key.isEmpty()) {
+			builder.append("[").append(key).append("]");
+		}
+		builder.append("[").append(value).append("]").append("\n");
+		return this;
+	}
+
+	public DiscordMarkdownBuilder checkLast() {
+		int length = builder.length();
+		String lastThreeChars = builder.substring(length - 3);
```

Comment:
Pretty sure this could crash with short messages. Probably won't since we preinitialize the thing with a longer string, but still.